packages/xo-server/src/api/pif.mjs

```diff
@@ -28,7 +28,7 @@ async function delete_({ pif }) {
     return
   }
 
-  await xapi.callAsync('PIF.destroy', pif._xapiRef)
+  await xapi.callAsync('PIF.forget', pif._xapiRef)
```

Comment:
From what I understand in the issue, we must do this only when `PIF.destroy` throws `PIF_IS_PHYSICAL`, am I wrong?

Reply:
That's right, initially I wanted to attempt a try/catch, but after discussing it with @pdonias we agreed that the topic had already been addressed in the issue and that we should simplify

Reply:
Perfect, thank you :slightly_smiling_face: